### PR TITLE
Feat: Replaces `CartItem` FSUI

### PR DESCRIPTION
**Pull request by renatamottam**
> `@faststore/ui` package needs to be updated after merge of https://github.com/vtex/faststore/pull/1614

## What's the purpose of this pull request?
- [x] Uses `CartItem` from the `faststore/ui`
- [x] Remove `CartItem` folder

## How does it work?
You should see the updated version of `CartItem`

## How to test it?
You can check the `CartItem` inside the `CartSidebar`

### CODE REVIEW

**Review by victorhmp — COMMENTED** [review]
LGTM

**Comment by hellofanny** [comment]
LGTM, but I found something strange with the gift section. Can you check it pls? :) 

Before | After|
|-|-|
image|image

File: src/components/cart/CartItem/CartItem.tsx [hunk 1]
```diff
@@ -89,68 +78,36 @@ function CartItem({ item, gift = false }: Props) {
...
-      className={styles.fsCartItem}
-      data-testid="cart-item"
-      data-sku={item.itemOffered.sku}
-      data-seller={item.seller.identifier}
+      price={{
+        value: item.price,
+        listPrice: item.listPrice,
+        formatter: useFormattedPrice,
+      }}
+      quantity={item.quantity}
```
Comment (hellofanny) [inline src/components/cart/CartItem/CartItem.tsx:95]: I think we are missing gift condition here.

Comment (renatamottam) [reply src/components/cart/CartItem/CartItem.tsx:95]: we should use the `Gift` component here, not the `CartItem`

Comment (hellofanny) [reply src/components/cart/CartItem/CartItem.tsx:95]: Done [here](https://github.com/vtex-sites/nextjs.store/pull/360/commits/9299cc0b5455b37cd6a1665bd98f065fe0e08003)

**Comment by hellofanny** [comment]
> LGTM, but I found something strange with the gift section. Can you check it pls? :)
> 
> Before	After
> image	image

Updated [here](https://sfj-fab50fa--nextjs.preview.vtex.app/aedle-vk1-headphone-99988211/p)

image

**Pull request merged**